app/models/image.rb

```diff
@@ -972,26 +972,81 @@ def self.update_copyright_holder(old_name, new_name, user)
     # This is orders of magnitude faster than doing via active-record.
     old_name = Image.connection.quote(old_name)
     new_name = Image.connection.quote(new_name)
-    data = Image.connection.select_rows(%(
-      SELECT id, YEAR(`when`), license_id FROM images
-      WHERE user_id = #{user.id} AND copyright_holder = #{old_name}
-    ))
+
+    select_manager = arel_select_copyright_holder(old_name, user)
+    # puts(select_manager.to_sql)
+    data = Image.connection.select_rows(select_manager.to_sql)
     return unless data.any?
```

Comment:
Notwithstanding my comment above, I think this part at very least would be equivalent in AR:

  data = Image.where(user: user, copyright_holder: old_name).pluck(:id, :when, :license_id)

Reply:
This looks right, but the original method only returns the "year" part of `:when`. Does that matter?

Reply:
@JoeCohen Here's what I found that works. It uses arel-helpers, which allow you to reference the table directly, using the class name and square brackets: 
```ruby
data = Image.where(user: user, copyright_holder: old_name).
       pluck(Image[:id], Image[:when].year, Image[:license_id])
```